app/controllers/press-releases/press-release.js

```diff
@@ -62,6 +62,7 @@ export default class PressReleasesPressReleaseController extends Controller {
 
   @task
   *publish(publicationDate) {
+    yield this.savePressRelease.perform();
```

Comment:
Ik zou misschien wel toevoegen dat de save enkel uitgevoerd wordt als press-release snapshot effectief `isDirty` is.